src/preferences/dialog/dlgprefreplaygaindlg.ui

```diff
@@ -246,6 +221,43 @@
     </widget>
    </item>
    <item>
+    <widget class="QGroupBox" name="groupBoxAnalysis">
+     <property name="title">
+      <string>Analysis</string>
+     </property>
+     <layout class="QVBoxLayout" name="verticalLayout_3">
+      <item>
+       <widget class="QRadioButton" name="radioButtonRG2">
+        <property name="text">
+         <string>ReplayGain 2.0 (EBU R 128)</string>
```

Comment:
Hm, I just read EBU R 128 and it doesn't describe any algorithm, it simply says use an average LUFS of -23 and use the algorithms described in ITU-R BS.1770. So this presentation is pretty confusing since this makes it look like you're selecting the EBU R 128 algorithm (which isn't a thing as far as I can tell). 

If we had an average LUFS selector that let you pick between RG2 (-18), EBU R 128 (-23), and ATSC A/85 (-24) then that would make more sense IMO but since we have a slider this wouldn't make sense. Perhaps the different averages could be mentioned in a tooltip on the slider?

What do you think of ```ReplayGain 2.0 (ITU-R BS.1770)``` 

* https://tech.ebu.ch/docs/r/r128.pdf
* https://www.itu.int/dms_pubrec/itu-r/rec/bs/R-REC-BS.1770-4-201510-I!!PDF-E.pdf